app/src/main/res/color/bottom_navigation_icon_fill_color_state.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="?colorPrimary" android:state_checked="true" />
+    <item android:color="@color/transparent" />
```

Comment:
Is there an `@android:color/transparent` we could use here?

Reply:
This file is now removed